### DISCUSSION TITLE
Looking for paid assistance with a simple MIMIC project

### DISCUSSION THREAD

**Discussion by ByronDrumheller** [MIT-LCP/mimic-code#1620] [MIMIC-General]
I am a physician clinical researcher with research focused on cardiac arrest resuscitation. I would like to use data from the MIMIC III Waveform Database Matched Subset to identify patients that suffered an in-ICU cardiac arrest with invasive arterial blood pressure monitoring in place at the time of arrest. I am interested in exporting the waveform signals during the arrest to Matlab and analyzing offline.

I have no experience with computer programming or database analysis at the level of MIMIC and do not have access to anyone at my institution with such expertise. I do have funding to be able to pay for this work, however. Thus, I am looking for assistance from anyone familiar with working with MIMIC to help with this project. This would only involve querying MIMIC and exporting the data; the Matlab analysis will not be involved. Payment is negotiable based upon the time required to perform the work, but at a minimum I would provide $5,000.00. Authorship and academic recognition on subsequent research would also be provided as appropriate.

Please contact me with interest at: byron.drumheller@gmail.com.

-Byron Drumheller, MD
Assistant Professor of Emergency Medicine
University of Pittsburgh School of Medicine

**Comment by alistairewj** [comment]
Hope you find someone! If you haven't had any luck feel free to ping me using the email in my GitHub profile and/or CV on my website.
